### ISSUE TITLE
if (type.equals(oldSku != null) {

### ISSUE THREAD

**Issue by virendave96** [dooboolab/react-native-iap#898]
### Version of react-native-iap
    "react-native-iap": "^4.4.0-rc.5",

### Version of react-native
    "react-native": "0.61.5",

### Platforms you faced the error (IOS or Android or both?)
Android

### Expected behavior
npx react-native run-android should run app in emulator.

### Actual behavior
react-native-iap\android\src\main\java\com\dooboolab\RNIap\RNIapModule.java:417: error: ')' expected
        if (type.equals(oldSku != null) {

### Tested environment (Emulator? Real Device?)
Emulator

### Steps to reproduce the behavior
Add the Library and run the project.

I update the code in node_modules and run its working but please solve it.  missing ')'

**Comment by nolanrbrady** [comment]
If you go to line 417 in the NPM package and make
```
        if (type.equals(oldSku != null) {
          builder.setOldSku(oldSku);
        }
```
into this
```
        if (type.equals(oldSku != null)) {
          builder.setOldSku(oldSku);
        }
```
If should run just fine. It looks like a simple typo that needs to be fixed with a PR.

**Comment by hyochan** [comment]
Fixes in `4.4.0-rc.7`.
